examples/02-Advanced-workflow.ipynb

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
<p>valid is the train dataset.</p><p> </p><p>We need to update cuDF to get_lib</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
switched to using `get_lib`

addressed the train/valid situation